make/Makefile.build.mk

```
@@ -25,6 +25,13 @@ build: go-check
 	${GO_BUILD_ENVVARS} ${GO} build \
 		-o ${GOPATH}/bin/kiali -ldflags "-X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"
 
+## build-linux-multi-arch: Build Kiali binary with arch suffix for multi-arch
+build-linux-multi-arch:
+	for arch in ${TARGET_ARCHS}; do \
+		${GO_BUILD_ENVVARS} ${GO} build \
+			-o ${GOPATH}/bin/kiali-$${arch} -ldflags "-X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"; \
```

Does this build the different arches? It looks like this just creates the same binary with different `-${arch}` in the names. GO_BUILD_ENVVARS will have GOARCH set to the same build host arch each time in the loop. Unless I misunderstand how this is working, I think you need to override GOARCH each time - something like this:
```suggestion
		${GO_BUILD_ENVVARS} GOARCH=${arch} ${GO} build \
			-o ${GOPATH}/bin/kiali-$${arch} -ldflags "-X main.version=${VERSION} -X main.commitHash=${COMMIT_HASH}"; \
```

---

Yes. i make a mistake here.
but without use $(MAKE). need to repeat GO_BUILD_ENVVARS `GOOS=linux GOARCH=$${arch} CGO_ENABLED=0`

updated.